tests/testthat/test-dataframe.R

```diff
@@ -166,6 +166,28 @@ test_that("Select with p$col", {
   expect_equal(z$columns, c("mpg", "hp"))
 })
 
+test_that("select with list of exprs", {
+  l_expr = list(pl$col("mpg"), pl$col("hp"))
+  l_expr2 = list(pl$col("mpg", "hp"))
+  l_expr3 = list(pl$col("mpg"))
+  l_expr4 = list(c("mpg", "hp"))
+  l_expr5 = list("mpg", "hp")
+
+  x1 = pl$DataFrame(mtcars)$select(l_expr)
+  x2 = pl$DataFrame(mtcars)$select(l_expr2)
+  x3 = pl$DataFrame(mtcars)$select(l_expr3, pl$col("hp"))
+  x4 = pl$DataFrame(mtcars)$select(pl$col("hp"), l_expr3)
+  x5 = pl$DataFrame(mtcars)$select(l_expr4)
+  x6 = pl$DataFrame(mtcars)$select(l_expr5)
+
+  expect_equal(x1$columns, c("mpg", "hp"))
+  expect_equal(x2$columns, c("mpg", "hp"))
+  expect_equal(x3$columns, c("mpg", "hp"))
+  expect_equal(x4$columns, c("mpg", "hp"))
+  expect_equal(x5$columns, c("mpg", "hp"))
+  expect_equal(x6$columns, c("mpg", "hp"))
+})
```

Comment:
I suppose these tests can be simplify by `patrick`.

Reply:
I've never used `patrick`, I find the tests hard to read (but feel free to modify those ofc)

Reply:
I don't mind patrick for large combinatorial tests. Looks neat. I find, that if a patrick test fails, and it is not immediately clear to me why, then I will spend more time than normal to debug the test.